android/expoview/src/main/java/versioned/host/exp/exponent/modules/universal/ExpoClientFilePermissionChecker.java

```diff
@@ -0,0 +1,12 @@
+package versioned.host.exp.exponent.modules.universal;
+
+import java.util.EnumSet;
+import expo.modules.filesystem.ExpokitFilePermissionChecker;
+import main.Permission;
+
+class ExpoClientFilePermissionChecker extends ExpokitFilePermissionChecker {
```

Comment:
How about:
```suggestion
class ScopedFilePermissionWielder extends FilePermissionWielder {
```

We tended to use `Scoped` prefix before. I also don't know if `Wielder` will work ok, that is probably question to native English speakers… `Manager` would be ok, but we have so many managers already… 😅